src/routes.tsx

```diff
@@ -425,5 +425,3 @@ export const routes: Route[] = [
     },
   },
 ]
-
-export const routePaths = routes.map((route) => route.path.toString())
```

Comment:
is this not imported anywhere?

Reply:
this was added so we can use it in useQuoteCartIdFromUrl but we are rolling all that logic back.